.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
 
-before_install:
-    - wget http://bit.ly/miniconda -O miniconda.sh
-    - bash miniconda.sh -b -p $HOME/miniconda
-    - export PATH="$HOME/miniconda/bin:$PATH"
-    - conda update --yes conda
-    - travis_retry conda create --yes -n test $CONDA --file requirements.txt
-    - source activate test
-    - travis_retry conda install --yes pip
-    - travis_retry pip install pytest
+# tdk: enable master when merged
+branches:
+  only:
+    - flexible-mesh
+#    - master
 
+sudo: false
 
 install:
-    - mkdir files
+  - if [[ "$CONDA_PYTHON_VERSION" == "2" ]]; then
+      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+    else
+      wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
```

Comment:
No need for this.  The tests are fast enough with conda downloading the python we need.  It just an unnecessary complexity to the travis configuration.